README.md

```diff
@@ -49,7 +49,7 @@ Install all the required tools and configurations using Microsoft's [windows-bui
 Install tools and configuration manually:
    * Install Visual C++ Build Environment: [Visual Studio Build Tools](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools)
    (using "Visual C++ build tools" workload) or [Visual Studio 2017 Community](https://visualstudio.microsoft.com/pl/thank-you-downloading-visual-studio/?sku=Community)
-   (using the "Desktop development with C++" workload)
+   (using the "Desktop development with C++" workload plus the components "Visual C++ compilers and libraries for ARM64" and "Visual C++ ATL for ARM64")
```

Comment:
This is not needed for users on Intel platforms, will probably even confuse some. Can you add this instead as a note to the bottom of the section, before or after the link to the guidelines? Something like "to compile on ARM, the following components are needed: ...".

Reply:
Sure thing. As much as I'd like everyone to build for all three Windows platforms (x86, x64, ARM64), I see what you mean about confusion. I'll move this to a side note.